.gitignore

```diff
@@ -4,3 +4,4 @@
 *.pyc
 *.sw[op]
 coverage.xml
+.DS_STORE
```

Comment:
Alternatively, since this is an OSX-specific file and not something inherent to this project, you could add this to your personal ignore file in `~/.gitignore`.
Then, those files will be ignored for any repo you work on.

CC: @tkim135